charts/redis-ha/templates/redis-ha-secret.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if or (regexFind "^.+@.+:.+" (toString .Values.restore.ssh.source)) (regexFind "^s3://.+|^S3://.+" (toString .Values.restore.s3.source)) }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "redis-ha.fullname" . }}-secret
+  namespace: {{ .Release.Namespace }}
+  labels:
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    app: {{ template "redis-ha.fullname" . }}
+type: Opaque
+data:
+{{- if regexFind "^.+@.+:.+" (toString .Values.restore.ssh.source) }}
```

Comment:
Wouldn't `{{ if .Values.restore.ssh.source }}' return nil since it's just `""` in the default chart? regexFind seems excessive

Reply:
i just want to make sure the ssh url is as it should be `user@server:path`